### PR TITLE
[Core 6.2]Doco for the receive pipeline completed event

### PR DESCRIPTION
**Pull request by andreasohlund**
That will ship with 6.2

https://github.com/Particular/NServiceBus/pull/4509

### CODE REVIEW

**Comment by andreasohlund** [comment]
Note that we plan to align the syntax for https://docs.particular.net/nservicebus/recoverability/subscribing-to-error-notifications in the near future

File: Snippets/Core/Core_6/Pipeline/OnReceivePipelineCompleted.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    class OnReceivePipelineCompleted
+    {
+        public OnReceivePipelineCompleted()
+        {
+            var endpointConfiguration = new EndpointConfiguration("MyEndpoint");
+
+            #region ReceivePipelineCompletedSubscription
+
+            endpointConfiguration.Pipeline.OnReceivePipelineCompleted(e =>
```
Comment (dvdstelt) [inline Snippets/Core/Core_6/Pipeline/OnReceivePipelineCompleted.cs]: When writing a feature, you would use `context.Pipeline.OnReceivePipelineCompleted(e =>`. Would that not be a better example to provide?

Comment (andreasohlund) [inline Snippets/Core/Core_6/Pipeline/OnReceivePipelineCompleted.cs]: Both are relevant, should I add a snippet showing a feature as well?

Comment (dvdstelt) [inline Snippets/Core/Core_6/Pipeline/OnReceivePipelineCompleted.cs]: Initially I wasn't sure, but I like to be explicit. Results in less questions from users. And the page doesn't contain the much information yet, so it's probably better to add it.

**Comment by andreasohlund** [comment]
Added snippet to show subscribing from a feature as well

**Pull request merged**